app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/components/AddEventAttendeeDialog.kt

```diff
@@ -37,70 +37,68 @@ fun AddEventAttendeeDialog(
     onAdd: () -> Unit,
     agendaItem: AgendaItemDetails.Event
 ) {
-    if(agendaItem.isAddingAttendee) {
-        var isFocused by remember { mutableStateOf(false) }
-        val focusRequester = remember { FocusRequester() }
+    var isFocused by remember { mutableStateOf(false) }
```

Comment:
Minor - whatever has focus will lose it when a configuration change occurs. Mostly, this isn’t a big concern, but you can work around this by using `rememberSaveable`